### PR TITLE
NO-ISSUE: add sshpass to dnf install on hypervisor

### PR DESCRIPTION
**Pull request by eggfoobar**
Some hosts lack `sshpass`, this adds `sshpass` to the dnf install line.

### CODE REVIEW

**Comment by dhellmann** [comment]
Good catch, thanks!

/approve
/lgtm

File: scripts/devenv-builder/create-vm.sh [hunk 1]
```diff
@@ -52,7 +52,7 @@ if [ "${SWAPSIZE}" -eq 0 ] ; then
     sed -i "s;^part swap;#part swap;" ${KICKSTART_FILE}
 fi
 
-sudo dnf install -y libvirt virt-manager virt-install virt-viewer libvirt-client qemu-kvm qemu-img
+sudo dnf install -y libvirt virt-manager virt-install virt-viewer libvirt-client qemu-kvm qemu-img sshpass
```
Comment (ggiguash) [inline scripts/devenv-builder/create-vm.sh:55]: Please, also update the https://github.com/openshift/microshift/blob/main/docs/devenv_setup.md#creating-vm doc.

Also, we have another `create-vm.sh` script+doc for image building, so we may want to make it consistent. See https://github.com/openshift/microshift/blob/main/scripts/image-builder/create-vm.sh

**Comment by ggiguash** [comment]
/retitle NO-ISSUE: add sshpass to dnf install on hypervisor

**Comment by ggiguash** [comment]
We need doc updates for this fix. Let's delay the merge.

/lgtm cancel

File: scripts/devenv-builder/create-vm.sh (hunk 1 above)
Comment (eggfoobar) [reply scripts/devenv-builder/create-vm.sh:55]: Sounds good, updated!

**Pull request merged**